PART_7/StrainTransmission/Iterate_and_job.py

```
@@ -0,0 +1,186 @@
+cd /groups/umcg-llnext/tmp01/umcg-sandreusanchez/Strain_transmission
+""".format(SGB = SGB)
+		Script_n = "scripts/{SGB}_StrainTransmLM.sh".format(SGB = SGB)
+		with open(Script_n,'w') as F: F.write(Job_setup + Command)
+		subprocess.call("sbatch "+Script_n, shell=True)
+		#print(Script_n)
+
+def Script3_mantelTest():
+	script = "4_MantelTest_Geography_vs_Distance.R"
+	for File in Path("NEXT_RAxML_distmats").glob("*.txt"):
+		SGB = File.stem.replace("_DistMat", "")
+		if Path("Mantel_distance/",SGB,".tsv").exists(): continue
```

Ruff (0.14.8)

Multiple statements on one line (colon)

(E701)